pyproject.toml

```diff
@@ -51,6 +50,25 @@ Funding = 'https://github.com/sponsors/samuelcolvin'
 Source = 'https://github.com/pydantic/pydantic'
 Changelog = 'https://pydantic-docs.helpmanual.io/changelog'
 
+[tool.hatch.metadata.hooks.fancy-pypi-readme]
+content-type = 'text/markdown'
+fragments = [
+    { path = 'README.md' },
+    { path = 'HISTORY.md' },
+]
+
+[[tool.hatch.metadata.hooks.fancy-pypi-readme.substitutions]]
+pattern = '#(\d+)'
```

Comment:
I think we need to require a space before the issue ID here like we do on usernames, or we could use a negative lookbehind to omit cases with `\w` directly before `#`. This caught us recently and the [second point on `v1.10.2`](https://pydantic-docs.helpmanual.io/changelog/) is currently messed up.

Reply:
Done!